numpy/__init__.pyi

```diff
@@ -737,6 +737,7 @@ class ufunc:
     @property
     def at(self) -> Any: ...
 
+abs: ufunc
```

Comment:
Considering it's an alias, why not just define it as such?
```suggestion
abs = absolute
```

Reply:
Also if we could add a really simple test case to `numpy/tests/typing/pass/ufuncs.py` that would be great!

Reply:
Thank you for reviewing. Ok, I'll add a test case. BTW, I found that `ufuncs.py` contain no other ufuncs except `sin`. Is it Ok to add a test case only for `abs`, or is it better to add all other ufuncs? (or parameterized test?)

Reply:
> Is it Ok to add a test case only for abs, or is it better to add all other ufuncs? (or parameterized test?)

I feel it's fine as it is now, especially since the (currently) rather broad `ufunc.__call__()` signature will make adding proper `fail` tests somewhat tricky.
Still, if you're interested feel free to add them, be it either in this pull request or another one.